pkg/swap/swap.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
```

Comment:
> bloats Alpine up to 12MB

Is it that much? Ouch. As it is, swap has to install `util-linux` because it needs `mkswap` and `swapon`. and eventually will need `cryptsetup`. I viewed bash as small potatoes comparatively. I wish I could get *just* `mkswap` and `swapon` without all of `util-linux`; if you know how, happy to take a pointer.

I guess I will try to rewrite to `sh`.